config/localTemplate.go

```diff
@@ -362,8 +362,9 @@ type Local struct {
 	// 0x01 (dnssecSRV) - validate SRV response
 	// 0x02 (dnssecRelayAddr) - validate relays' names to addresses resolution
 	// 0x04 (dnssecTelemetryAddr) - validate telemetry and metrics names to addresses resolution
+	// 0x08 (dnssecTXT) - validate TXT response
 	// ...
-	DNSSecurityFlags uint32 `version[6]:"1"`
+	DNSSecurityFlags uint32 `version[6]:"1" version[34]:"9"`
```

Comment:
So this is validate SRV response and validate TXT response right?

Reply:
correct, the original code shortcut it and used SRV flag for TXT validation.